theme-contract/src/main/java/com/orange/ouds/theme/OudsThemeContract.kt

```diff
@@ -57,6 +58,11 @@ interface OudsThemeContract : Parcelable {
      */
     val borderTokens: OudsBorderSemanticTokens
 
+    /**
+     * Effect semantic tokens values used in the theme.
+     */
+    val effectTokens: OudsEffectSemanticTokens
```

Comment:
Out of curiosity, do you know what are these tokens for?

Reply:
There is only one token for the moment to implement an effect on drag